stable/pachyderm/Chart.yaml

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: v1
+description: >
+  Pachyderm is a large-scale container-based workflow engine that offers complete version control for data and reproducible data processing
+name: pachyderm
+keywords:
+  - pachyderm
+  - workflows
+  - reproducibility
+  - distributed
+  - processing
+version: 0.1.0
+home: "https://pachyderm.io"
```

Comment:
would it make sense to add this icon? https://avatars2.githubusercontent.com/u/10432478?s=200&v=4